tensorflow-core/tensorflow-core-api/src/test/java/org/tensorflow/SessionTest.java

```diff
@@ -244,9 +249,54 @@ public void saveAndRestore() throws IOException  {
 
     // Cleanup test dir
     Files.walk(testFolder)
-            .sorted(Comparator.reverseOrder())
-            .map(Path::toFile)
-            .forEach(File::delete);
+        .sorted(Comparator.reverseOrder())
+        .map(Path::toFile)
+        .forEach(File::delete);
+  }
+
+  @Test
+  public static void testFetchVariable() {
```

Comment:
Might be an idea to have a test that loads in a GraphDef from python which has a resource variable in it. That way we're definitely testing the most common way to exercise that codepath.